### PR TITLE
Fix "Creating a Cross-Platform Console App with DNX"

### PR DESCRIPTION
**Pull request by bgrainger**
This addresses comments left at http://docs.asp.net/en/latest/dnx/console.html#comment-2299237347

### CODE REVIEW

File: common/samples/ConsoleApp1/src/ConsoleApp1/Program.cs [hunk 1]
```diff
@@ -1,7 +1,4 @@
 ﻿using System;
-using System.Collections.Generic;
```
Comment (danroth27) [inline common/samples/ConsoleApp1/src/ConsoleApp1/Program.cs]: The common samples are snap shots of the Visual Studio project templates. We prefer that they don't diverge. If you want to propose changes to the project templates you can contributed to https://github.com/aspnet/templates.

**Comment by danroth27** [comment]
The RST changes look good, but the common samples should be left unchanged.

File: common/samples/ConsoleApp1/src/ConsoleApp1/Program.cs (hunk 1 above)
Comment (bgrainger) [inline common/samples/ConsoleApp1/src/ConsoleApp1/Program.cs]: The problem with the documentation is that it says "Then, create a file with these contents and save it as Program.cs in the ConsoleApp1 folder" but the contents of the file aren't what the article wants them to be.

Would it be better to create a new sample with the right contents (I'm guessing no) or remove the `literalinclude` and simply put the designed source code directly in the RST (I'd have to look up the syntax for this)?

Comment (danroth27) [inline common/samples/ConsoleApp1/src/ConsoleApp1/Program.cs]: Yeah, let's go ahead and add a sample specific to this doc then. The common samples should just stay as they are.

**Comment by rachelappel** [comment]
Merged into docs. Thank you for your contribution! f1800d42dc3fc994e227caa8d6ba87169a96ef9b

**Pull request closed without merging**